src/org/traccar/api/resource/UserResource.java

```diff
@@ -50,6 +53,18 @@ public Response add(User entity) throws SQLException {
         if (!Context.getPermissionsManager().isAdmin(getUserId())) {
             Context.getPermissionsManager().checkRegistration(getUserId());
             Context.getPermissionsManager().checkUserUpdate(getUserId(), new User(), entity);
+            Config config = Context.getConfig();
+            int deviceLimit = config.getInteger("default.deviceLimit");
+            int expirationDuration = config.getInteger("default.expirationDuration");
+            if (deviceLimit > 0) {
+              entity.setDeviceLimit(deviceLimit);
+            }
+            if (expirationDuration > 0) {
+              Calendar cal = Calendar.getInstance();
```

Comment:
Yes, that's what I meant. Should be very similar to this:

[DataManager.java#L306](https://github.com/tananaev/traccar/blob/master/src/org/traccar/database/DataManager.java#L306)